src/services/messages.ts

```diff
@@ -1,19 +1,22 @@
+import { doorCode } from '../framework/environment.js';
+
 export const welcome = 'Welcome to the gate!👋🏻';
 
 export const notAllowed =
-  'Looks like you are not allowed to open the gate⛔🙅🏻\nTo request access, try using /request_access\nTo check if you are allowed to open the gate, try using /check_authorization';
+  'Looks like you are not an allowed user⛔🙅🏻\nTo request access, use /request_access\nTo check if you are have been approved, use /check_authorization';
 
-export const allowed = 'You are allowed to open the gate!🥳🎉\nTry using /open to open the gate';
+export const allowed = 'You are allowed to open the gate!🥳🎉';
 
 export const opening = 'Opening...🔓';
 
 export const failedToOpen = 'Failed to open the gate⚠️';
 
-export const help =
-  'Try using /open to open the gate\nTo request access, try using /request_access\nTo check if you are allowed to open the gate, try using /check_authorization';
+export const helpAllowed = 'Use /open to open the gate🔑\nTo check the door code, use /door_code 🔢';
 
 export const alreadyAllowed = 'You are already allowed to open the gate!🥳🎉';
 
+export const requestSent = 'Request sent!📨';
+
 export const accessDenied = 'Your access to this bot has been denied⛔🙅🏻';
 
-export const getDoorCode = (code: string) => `The door code is 🔢\n${code}`;
+export const doorCodeDetails = doorCode ? `The door code is 🔢\n${doorCode}` : 'The door code is not set.';
```

Comment:
If you remove the action, why do you need this?

Reply:
Yeah you can't really get here if it's disabled.
I just want the type to stay string